moped-editor/src/views/projects/projectView/ProjectComponentsBaseMap.js

```diff
@@ -85,62 +107,67 @@ const ProjectComponentsBaseMap = ({
     setBasemapSpeedDialOpen(true);
   };
 
-  return containerRef && containerRef.current
-    ? ReactDOM.createPortal(
```

Comment:
I converted this from a portal to a plain old component because I was having trouble piecing together the container the portal tied to, its ref, and this component. If anyone disagrees, I'll change it back.

Reply:
👍 have never personally been able to get a portal to work, so i'm happy to see it go 🚀